tests/bluetooth/bsim_bt/bsim_test_eatt_notif/src/client_test.c

```diff
@@ -153,7 +153,24 @@ static void gatt_discover(void)
 BT_GATT_SERVICE_DEFINE(g_svc,
 	BT_GATT_PRIMARY_SERVICE(TEST_SERVICE_UUID),
 	BT_GATT_CHARACTERISTIC(TEST_CHRC_UUID, BT_GATT_CHRC_NOTIFY,
-			       0x00, NULL, NULL, NULL));
+			       BT_GATT_PERM_READ, NULL, NULL, NULL),
+	BT_GATT_CCC(NULL,
+		    BT_GATT_PERM_READ | BT_GATT_PERM_WRITE));
+
+static void subscribe_client(const struct bt_conn *conn, const struct bt_gatt_attr *attr)
+{
+	while (bt_uuid_cmp(attr->uuid, BT_UUID_GATT_CCC) != 0) {
+		attr = bt_gatt_attr_next(attr);
+	}
+
+	/* Subscribe client without doing the actual discovery on the client itself */
+	struct _bt_gatt_ccc *ccc = attr->user_data;
+	struct bt_gatt_ccc_cfg *cfg = &ccc->cfg[0];
+
+	cfg->id = bt_conn_index(conn);
+	ccc->cfg[0].value = BT_GATT_CCC_NOTIFY;
```

Comment:
```suggestion
	cfg->value = BT_GATT_CCC_NOTIFY;
```

Reply:
I think the `cfg[0]` makes it clearer that `cfg` is an array of configurations for each connection. I'm fine with your syntax though.

Reply:
That's fair :) Personally I always dislike the `&ccc->cfg[0];` syntax compared to `ccc->cfg;`, but that's just a personal preference :) Either way works, and if you prefer the other I'm OK with that.